packages/manager/modules/sms/src/sms/sms/compose/telecom-sms-sms-compose.controller.js

```diff
@@ -295,7 +295,9 @@ export default class {
     this.displaySenderCustomizationAdvice =
       isRealNumber || this.sms.sender === SMS_COMPOSE.shortNumber;
     this.canHaveSTOPAnswer =
-      !isRealNumber && this.sms.sender !== SMS_COMPOSE.shortNumber;
+      !isRealNumber &&
+      this.sms.sender !== SMS_COMPOSE.shortNumber &&
+      this.user.ovhSubsidiary === 'FR';
```

Comment:
```suggestion
      this.displayCommercialClause();
```
so you handle logic at one place